src/_Common/Functions.cs

```diff
@@ -105,6 +105,36 @@ internal static TimeSpan ToTimeSpan(this PeriodSize periodSize)
             };
         }
 
+        internal static PeriodSize? ToPeriodSize(this TimeSpan timeSpan)
```

Comment:
`internal` will not be accessible by library users, so you'll have to make this `public`

Reply:
can also public ToTimeSpan and RoundDown method?

Reply:
I'm not a fan of exposing all these other internals if we can avoid it.  If it's necessary, go ahead and make them `public`.